scripts/common/prompts.sh

```diff
@@ -2,36 +2,18 @@
 # Gather any additional information required from the user that could not be discovered and was not
 # passed with a flag
 
-function prompts_can_prompt() {
-    # Need the TTY to accept input and stdout to display
-    # Prompts when running the script through the terminal but not as a subshell
-    if [ -t 1 ] && [ -c /dev/tty ]; then
-        return 0
-    fi
-    return 1
-}
-
 function prompt() {
-    if ! prompts_can_prompt ; then
-        bail "Cannot prompt, shell is not interactive"
-    fi
-
     set +e
-    read PROMPT_RESULT < /dev/tty
+    read -p "(y/n)?" PROMPT_RESULT
     set -e
 }
 
 function confirmY() {
-    printf "(Y/n) "
     if [ "$ASSUME_YES" = "1" ]; then
+        printf "(Y/n) "
```

Comment:
Why is this moving inside the conditional. This is prompting the user with options.

Reply:
Because we are printing it with the read already `https://github.com/replicatedhq/kURL/pull/3891/files#diff-04b0e97fb7895ebf075d4f880f40b32b7fc78978539ff2932c492400967feec8R7` so we do not need print it unless we are in the scenario where we assume yes.